File: IgnatiusMobile/app/src/main/java/com/patrick/ignatiusMobile/MainActivity.java
package com.patrick.ignatiusMobile;

import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import android.widget.AdapterView;
import android.content.Intent;
import android.widget.ListView;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);
        AdapterView.OnItemClickListener itemClickListener = new
                AdapterView.OnItemClickListener() {
                    @Override
                    // Main Menu setup
                    public void onItemClick(AdapterView<?> adapterView, View view,
                                            int position, long id) {
                        // Start Quiz menu
                        if (position == 0) {
                            Intent intent = new Intent(MainActivity.this,
                                   Quizme.class);
                            startActivity(intent);
                        }


                        // Go to add questions menu
                        if (position == 1) {
                            Intent intent = new Intent(MainActivity.this,
                                   AddQuestions.class);
                            startActivity(intent);
                        }


                        // show/delete questions

                        if (position == 2) {
                            Intent intent = new Intent(MainActivity.this,
                                    EditQuestions.class);
                            startActivity(intent);
                        }

                        if(position == 3)
                        {
                            Intent intent = new Intent(MainActivity.this,
                                    CSVOptions.class);
                            startActivity(intent);
                        }
                        // Exit Application

                        if (position == 4) {
                            finish();
                            System.exit(0);
                        }
                    }
                };
        ListView listView = findViewById(R.id.list_options);
        listView.setOnItemClickListener(itemClickListener);
    }


}

File: IgnatiusMobile/app/src/main/java/com/patrick/ignatiusMobile/MainActivity.kt
package com.patrick.ignatiusMobile

import android.content.Intent
import android.os.Bundle
import android.widget.AdapterView.OnItemClickListener
import android.widget.ListView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val itemClickListener = OnItemClickListener { adapterView, view, position, id ->
            // Main Menu setup
            // Start Quiz menu
            if (position == 0) {
                val intent = Intent(this@MainActivity,
                        Quizme::class.java)
                startActivity(intent)
            }


            // Go to add questions menu
            if (position == 1) {
                val intent = Intent(this@MainActivity,
                        AddQuestions::class.java)
                startActivity(intent)
            }


            // show/delete questions
            if (position == 2) {
                val intent = Intent(this@MainActivity,
                        EditQuestions::class.java)
                startActivity(intent)
            }

            // Exit Application
            if (position == 3) {
                finish()
                System.exit(0)
            }
        }
        val listView = findViewById<ListView>(R.id.list_options)
        listView.onItemClickListener = itemClickListener
    }
}